packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -219,6 +220,9 @@ export default defineComponent({
         this.isSharedWithOthersLocation ||
         this.isSharedViaLinkLocation
       )
+    },
+    spaceResource() {
+      return this.spaces.find((s) => s.id === this.highlightedFile.id)
```

Comment:
Again it makes it so much more complicated that the sidebar sometimes uses the `highlightedFile` for showing and mutating resources, sometimes we use the provided `displayedItem`, and now sometimes the `spaceResource`. It would be so nice to have that streamlined to one provided prop (either via `provide` or via `props`).

Reply:
fully agree 😕 I'm in favour of `provide/inject` for the data context of the sidebar. With props it'd be a pretty heavy cascade in some cases.